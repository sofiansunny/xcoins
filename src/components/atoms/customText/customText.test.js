import React from 'react'
import {shallow} from 'enzyme'
import toJson from 'enzyme-to-json'
import CustomText from '.'
describe('SomeComponent component', () => {
  it('Shallow rendering', () => {
    const props = {
      style: {},
      textType: 'regular' | 'bold' | 'light',
    }
    const wrapper = shallow(<CustomText {...props} />)
    expect(toJson(wrapper)).toMatchSnapshot()
  })
})
