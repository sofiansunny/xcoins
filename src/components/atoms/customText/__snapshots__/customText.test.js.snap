// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`SomeComponent component Shallow rendering 1`] = `
<Text
  style={
    Array [
      Object {
        "fontFamily": "Lato-Regular",
      },
      Object {},
    ]
  }
/>
`;
